src/modules/pages/components/RouteLayouts/UserNavigation.css

```
@@ -96,7 +96,7 @@
 }
 
 .walletAddress {
-  padding: 2px 9px 3px;
+  padding: 3px 9px 2px;
```

For me it looks "aligned" if I change the top padding to `0`

```suggestion
  padding: 0 9px 2px;
```